js/plugins/trimTimeline.js

```diff
@@ -1,33 +1,35 @@
+var trimClinicalTimeline = new clinicalTimelinePlugin("trimTimeline", "Trim Timeline");
```

Comment:
It's not entirely clear to me which inheritance pattern you using for the plugins now: http://davidshariff.com/blog/javascript-inheritance-patterns/. I would advise to pick something that's common, but perhaps this is a pattern I'm not familiar with.